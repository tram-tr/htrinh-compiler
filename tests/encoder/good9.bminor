"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum tristique\nquam ut ante efficitur iaculis. Nunc sed tortor ut purus rhoncus bibendum.\nFusce id dolor nulla. Suspendisse laoreet facilisis est, eu posuere mi\nsagittis id."