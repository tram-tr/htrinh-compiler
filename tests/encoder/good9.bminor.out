decoded: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum tristique
quam ut ante efficitur iaculis. Nunc sed tortor ut purus rhoncus bibendum.
Fusce id dolor nulla. Suspendisse laoreet facilisis est, eu posuere mi
sagittis id.
encoded: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum tristique\nquam ut ante efficitur iaculis. Nunc sed tortor ut purus rhoncus bibendum.\nFusce id dolor nulla. Suspendisse laoreet facilisis est, eu posuere mi\nsagittis id."
encoding successful.
